qiling/os/windows/dlls/kernel32/tlhelp32.py

```diff
@@ -27,7 +28,7 @@ def hook_CreateToolhelp32Snapshot(ql, address, params):
     # TODO thinking about implementing an handler, gonna see if is really necessary
     flag = params["dwFlags"]
     if flag == TH32CS_SNAPPROCESS:
-       ql.dprint(D_RPRT, "[=] The sample is checking every process!")
+        ql.dprint(D_RPRT, "[=] The sample is checking every process!")
```

Comment:
Binary .. targeted process or somthing. the word sample is very generic to malware

Reply:
c9e25f5